tools/win_compile_sanity_check.ps1

```diff
@@ -0,0 +1,28 @@
+# Script to test if a small target builds quickly on windows as expected.
+
+# Define the timeout value in seconds
+$timeout = 120
+
+# Start a timer
+$start = Get-Date
+
+# Run the command
+$command = "python .\cobalt\build\gn.py .\out\win32_test -p win-win32 -C devel ; ninja -C .\out\win32_test\ eztime_test"
+Invoke-Expression $command
+
+# Get the end time
+$end = Get-Date
+
+# Calculate the execution time
+$executionTime = $end - $start
+$executionTimeinSecs = $executionTime.seconds
+Write-Output "Execution time in secs is $executionTimeinSecs"
+
+# Check if the execution time is greater than the timeout value
+if ($executionTime.seconds -gt $timeout) {
+    # Throw an error
+    throw "The command took more than $timeout seconds to run."

```

Comment:
One more thought: consider putting in a go/link or some other reference to troubleshoot the issue here. Given this is open source, ideally this should go to an .md doc that is in the repo and helps with it. However, an internal go-link like `go/cobalt-windows-build#speed` or as a last resort bug link would do as well.

Reply:
added bug link